spec/unit/library/filter/replace_between_spec.rb

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright:: 2018 Sous Chefs
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require 'rspec_helper'
+include Line
+
+describe 'replace_between method' do
+  before(:each) do
+    @filt = Line::Filter.new
+    @filt.eol = "\n"
```

Comment:
```suggestion
    @filter.eol = "\n"
```

Reply:
Can we be generous with the variable names here? :)

Reply:
I've been coding to much golang lately.